experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts

```diff
@@ -83,9 +91,8 @@ export interface SharedPropertyTreeOptions {
  * in the total order)
  */
 export class SharedPropertyTree extends SharedObject {
-	// Initial state of the PRNG.  Must not be zero.  (See `advance()` below for details.)
-	tipView: SerializedChangeSet = {};
-	remoteTipView: SerializedChangeSet = {};
+	tipView: CommitNode = { changeSet: {}, metadata: {}};
```

Comment:
Why are these views commit nodes? They don't point to a specific change. Instead it is the squased result of all changes. Metadata doesn't really apply here.

Reply:
Yeah, I wasnt sure about this either, I wanted to keep the metadata close but we can def. just fetch it from the last entry of the change arrays